### ISSUE TITLE
negated props?

### ISSUE THREAD

**Issue by oyeanuj** [quentin-sommer/react-useragent#5]
Hi @quentin-sommer, firstly, thank you for creating this useful library! 

In a particular use-case, I was looking to target only a mobile phones, but not tablets. Unfortunately, I noticed that the `mobile` prop is set to `true` on both mobile and tablet devices.

So, in such a case, what do you recommend? Would you consider adding a `false` value such that the code below would target only say mobile phones:
```jsx
<UserAgent mobile tablet = { false } /> 
```

If so, that would be useful for people who want to target only certain browsers on certain mobile devices as this would provide the desired flexibility?

Thank you!

**Comment by quentin-sommer** [comment]
Hi, Thanks for using the library!

I'm afraid I'm the one responsible for mobile and tablet being both matched at once, it's happening because the mobile prop is optimistic and matches mobile + Android or iOS detection 
```
uaResults.mobile =
  uaResults.android ||
  uaResults.ios     ||
  uaParser.getDevice().type === 'mobile'

uaResults.tablet = uaParser.getDevice().type === 'tablet'
```

Will it work for you I i just disable this part?
```
uaResults.android ||
uaResults.ios     ||
```

It will ensure that only one of `mobile` or `tablet` prop is true

**Comment by oyeanuj** [comment]
@quentin-sommer I think that should work. I think the keeping the OS check separate from device type would help reduce false positives and provide flexibility for those who want those checks for their usecase.

**Comment by quentin-sommer** [comment]
Fixed starting `1.3.0`!
https://github.com/quentin-sommer/react-useragent/tree/v1.3.0
